syntax.Rmd

```diff
@@ -465,26 +465,49 @@ do_something_very_complicated("that", requires, many, arguments,
 
 As described under [Named arguments](#argument-names), you can omit the argument names
 for very common arguments (i.e. for arguments that are used in almost every
-invocation of the function). Short unnamed arguments can also go on the same
-line as the function name, even if the whole function call spans multiple lines.
+invocation of the function). If this introduces a large disparity between the line lengths, you may want to supply names anyway:
 
 ```{r, eval = FALSE}
-map(x, f,
+# Good
+map(
+  x,
+  f,
+  extra_argument_a = 10,
```

Comment:
aside: is `map()` the best function to use here? I seem to recall there's some movement on whether this should be written as `map(x, \(xi) f(xi, extra_argument_a = 10, extra_argument_b = ...))`

Reply:
Oh yeah, that's a good point.

Reply:
Agree about not showing an example where we pass extra args through the `...` of `map()`, which we are slowly discouraging